app/src/main/java/org/simple/clinic/summary/teleconsultation/messagebuilder/LongTeleconsultMessageBuilder_Old.kt

```diff
@@ -25,28 +26,29 @@ class LongTeleconsultMessageBuilder_Old @Inject constructor(
    * for the recipient.
    */
   override fun message(patientTeleconsultationInfo: PatientTeleconsultationInfo): String {
-    val message = StringBuilder("*${patientTeleconsultationInfo.facility.name}* teleconsult request for:")
+    val longPatientTeleconsultationInfo = patientTeleconsultationInfo as PatientTeleconsultationInfoLong
```

Comment:
We could make this more explicit using generics. Since this class is eventually going to get deleted, I think it's fine to leave it as is for now and take a call later.